script/deploy/tron/deploy-and-register-near-intents-facet.ts

```
@@ -0,0 +1,223 @@
+  getRPCEnvVarName,
+  getEnvironment,
+  getContractAddress,
+  checkExistingDeployment,
+  confirmDeployment,
+  printDeploymentSummary,
+  displayNetworkInfo,
+  displayRegistrationInfo,
+  getFacetSelectors,
+} from '../../utils/utils'
+import { proposeDiamondCut } from '../shared/propose-diamond-cut'
+import { getContractVersion } from '../shared/getContractVersion'
```

ESLint

[error] `../shared/getContractVersion` import should occur before import of `../shared/propose-diamond-cut`

(import/order)